x/evm/keeper/utils.go

```diff
@@ -11,19 +11,44 @@ import (
 
 	evmtypes "github.com/evmos/ethermint/x/evm/types"
 
+	"github.com/ethereum/go-ethereum/common"
 	"github.com/ethereum/go-ethereum/core"
 	ethtypes "github.com/ethereum/go-ethereum/core/types"
 )
 
-// DeductTxCostsFromUserBalance it calculates the tx costs and deducts the fees
-func (k Keeper) DeductTxCostsFromUserBalance(
+// DeductTxCostsFromUserBalance deducts the fees from the user balance. Returns an
+// error if the specified sender address does not exist or the account balance is not sufficient.
+//
+// TODO: Currently, Goland raises the warning that there are methods defined on both *keeper and keeper, which is not recommended: https://go.dev/doc/faq#methods_on_values_or_pointers
```

Comment:
This warning is raised by Goland. Should we address this or just ignore this warning? Link: https://go.dev/doc/faq#methods_on_values_or_pointers

Reply:
@MalteHerrmann can you remove the TODO?